src/gcs.rs

```diff
@@ -642,20 +705,53 @@ mod tests {
 
         let private_bucket = test_bucket();
         let filename = "get_private_object.txt";
+        let file_length: u64 = 769;
 
         let url = format!(
             "https://www.googleapis.com/storage/v1/b/{}/o/{}",
             private_bucket, filename
         );
 
         let object_url = Url::parse(&url).unwrap();
-        let object: Object = get_object(object_url).await.unwrap();
+        let object = match get_object(object_url).await {
+            Ok(o) => o,
```

Comment:
I'm also opening to approaching this by writing fresh test data in all cases, never relying upon or using an existing file of this name. I don't know if there's importance to using a particular file for the maintainer(s), so that is why I took the create-if-not-found tack.

Reply:
Yeah, the motivation here was development bringup: when I had only written "read-only" (which gets you a long way!), I did this with "I'll just have an existing file" so that the test only exercised the read path.

Now that we've gotten past that, I think it makes sense (for now) to always create a "get_private_object.txt". I kinda wish there was an obvious / easy way to mock the GCS response here, so that the test would still be "read only", but meh.